roadmap.md

```diff
@@ -0,0 +1,38 @@
+Roadmap Zombienet v2
+
+## Infra
+- Chaos testing, add examples and explore possibilities in `native` and `podman` provider
+- Add `docker` provider
+- Add `nomad` provider
+- Create [helm chart](https://helm.sh/docs/topics/charts/) to allow other use zombienet in k8s
+- Auth system to not use k8s users
+- Create GitHub Action and publish in marketplace (wip)
+- Rename `@paritytech/zombienet` npm package to `zombienet`. Keep all zombienet modules under `@zombienet/*` org
+
+## Internal teams
+- Add more teams (wip)
+
+## UI
+- Create UI to create `.zndls` and `network` files.
+- Improve VSCode extension (grammar/snippets/syntax highlighting/file validations) ([repo](https://github.com/paritytech/zombienet-vscode-extension))
+- Create UI app (desktop) to run zombienet without the need of terminal.
+
+## Registry
+- Create decorators registry and allow override by paras (wip)
+- Explore how to get info from paras.
+
+## Functional tasks
+- Add subxt integration, allow to compile/run on the fly
+- Move parser to pest (wip)
+- Detach phases and use JSON to communicate instead of `paths`
+- Add relative values assertions (for metrics/scripts)
+- Allow to define nodes that are not started in the launching phase and can be started by the test-runner
+- Allow to define `race` assertions
+- Rust integration -> Create multiples libs (crates)
+- Explore backchannel use case
+- Add support to run test agains a running network (wip)
+- Add more CLI subcommands
+- Add js/subxt snippets ready to use in assertions (e.g transfers)
+- Add XCM support in built-in assertions
+- Add support to start from a live network (fork-off) [check subalfred]
```

Comment:
From our experience (already doing fork-off Moonbeam chains):
This will require chain-specific actions (changing the authoring values, changing council, changing balance....). So a proper way to interact with the raw state is needed for each case.
It also requires to deal with very large database (or exported state json file) (Moonbeam > 5GB), which is not often easily done.

Reply:
We should perhaps file sub-issues to all roadmap items for further discussion once finalized. 

Certainly, this is a desirable feature. The main motivation I see is to test node or runtime changes against live network states as a pre-release check. This does require some ability to interact with raw state. To start from a live network state, the two major things are to allow swapping out the runtime (optional) and to allow modification of the state.

Reply:
> We should perhaps file sub-issues to all roadmap items for further discussion once finalized.
> 
> Certainly, this is a desirable feature. The main motivation I see is to test node or runtime changes against live network states as a pre-release check. This does require some ability to interact with raw state. To start from a live network state, the two major things are to allow swapping out the runtime (optional) and to allow modification of the state.

Thanks for your feedback @rphmeier! I will create a couple of umbrella issues to capture the vision/plan for the public roadmap and individual issues for items.

Thanks!